app/javascript/components/new-product-form.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div>
+    <form action="/create" method="POST" class="new-product-form" enctype=multipart/form-data >
+      <input type="hidden" name="authenticity_token" :value="csrfToken" autocomplete="off">
+      <div class="new-product-form__info">
+        <div class="product-input">
+          <div class="product-input__label">Nombre</div>
+          <input type="text" class="product-input__box" name="name">
+        </div>
+        <div class="product-input">
+          <div class="product-input__label">Precio</div>
+          <input type="text" class="product-input__box" name="price"/>
+        </div>
+        <div class="product-input">
+          <div class="product-input__label">Link</div>
+          <input type="text" class="product-input__box" name="link"/>
+        </div>
+        <div class="product-input">
+          <input type="submit" class="product-input__btn" value="Agregar" >
+        </div>
+      </div>
+      <div class="new-product-form__image">
+        <div class="image-preview" v-if="imageData.length > 0" >
+          <img class="image-preview__preview" :src="imageData">
+        </div>
+        <div class="image-preview" v-else>
```

Comment:
Attribute "else" should be on a new line                vue/max-attributes-per-line